co.electriccoin.swiftlint.yml

```diff
@@ -0,0 +1,110 @@
+# This SwiftLint file is based on this great guidelines. 
+# https://github.com/raywenderlich/swift-style-guide
+
+excluded:
+  - ${PWD}/Carthage
+  - ${PWD}/Pods
+  - ${PWD}/DerivedData
+  - ${PWD}/xctemplates
+
+disabled_rules:
+  - discarded_notification_center_observer
+  - notification_center_detachment
+  - orphaned_doc_comment
+  - todo
+  - unused_capture_list
+
+opt_in_rules:
+  - array_init
+  - attributes
+  - closure_end_indentation
+  - closure_spacing
+  - collection_alignment
+  - colon # promote to error
+  - convenience_type
+  - discouraged_object_literal
+  - empty_collection_literal
+  - empty_count
+  - empty_string
+  - enum_case_associated_values_count
+  - fatal_error_message
+  - first_where
+  - force_unwrapping
+  - implicitly_unwrapped_optional
+  - indentation_width
+  - last_where
+  - legacy_random
+  - literal_expression_end_indentation
+  - multiline_arguments
+  - multiline_function_chains
+  - multiline_literal_brackets
+  - multiline_parameters
+  - multiline_parameters_brackets
+  - operator_usage_whitespace
+  - overridden_super_call
+  - pattern_matching_keywords
+  - prefer_self_type_over_type_of_self
+  - redundant_nil_coalescing
+  - redundant_type_annotation
+  - strict_fileprivate
+  - toggle_bool
+  # - trailing_closure # weird in SwiftUI
+  - unneeded_parentheses_in_closure_argument
+  - unused_import
+  - vertical_whitespace_closing_braces
+  - vertical_whitespace_opening_braces
+  - yoda_condition
+
+
+custom_rules:
```

Comment:
A few more custom rules to consider adding:

string_concatenation:
    included: ".*\\.swift"
    excluded: ".*Test\\.swift"
    name: "String Concatenation"
    regex:  " \\+ \"|\" \\+ |\\+= \""
    message: "Please use string interpolation instead of concatenation"
    severity: error
    
    
print_function_usage:
    included: ".*\\.swift"
    excluded: ".*Test\\.swift"
    name: "Swift `print()` or `debugPrint()` should not be used in App Code"
    regex: "print\\(|debugPrint\\("
    message: "The Swift `print()` or `debugPrint()` functions should not be used."
    severity: warning

nslog_function_usage:
    included: ".*\\.swift"
    excluded: ".*Test\\.swift"
    name: "Swift `NSLog()` should not be used in App Code"
    regex: "NSLog\\("
    message: "The swift `NSLog` function should not be used."
    severity: error